config/default.yml

```
@@ -153,6 +153,12 @@ Rails/Date:
     - strict
     - flexible
 
+Rails/DefaultScope:
+  Description: 'Looks for using of `default_scope`.'
+  Reference: 'https://andycroll.com/ruby/dont-use-default-scope/'
+  Enabled: true
```

It may be an indicator when creating a new Rails application, but it is difficult to remove `default_scope` from an existing Rails application that already has `default_scope` installed. It is better to disable it by default and enable it for users who need it.

```suggestion
  Enabled: false
```

---

Seems to me most new cops should be make `pending`, as in the main gem.